chart/templates/deployment.yaml

```diff
@@ -358,3 +358,15 @@ spec:
             sizeLimit: "{{ .Values.osquery.logging.filesystem.volumeSize }}"
         {{- end }}
       {{- end }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
```

Comment:
There's already an `affinity` set up (see [L32-L43](https://github.com/fleetdm/fleet/blob/db3c73d619ff50dddb2ea4ead828340bcf80800a/charts/fleet/templates/deployment.yaml#L32-L43)) with a pod anti-affinity to ensure the pods are distributed across the nodes, and we should not have duplicate `affinity` keys.

Is there a planned use for this? The comment you included in the values file indicates a leader-follower model which Fleet does not have.

Reply:
i've moved that vars to `values.yml`